source/_integrations/amcrest.markdown

```diff
@@ -421,6 +423,15 @@ amcrest:
     stream_source: snapshot
     sensors:
       - ptz_preset
+
+  # Add doorbell camera device
+  - host: IP_ADDRESS_CAMERA_3
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+    name: Front Door Camera
+    sensors:
+      - motion_detected
+      - doorbell
```

Comment:
This PR violated ADR-0007 and ADR-0010

- https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md
- https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration still uses a platform configuration and adding a sensor here affects YAML options for this integration. Additionally, it probably violates ADR-0003 as well, as this could be considered a monitored condition (which is also no longer allowed).

The parent PR needs to be reverted.